compiler/rustc_query_system/src/dep_graph/serialized.rs

```diff
@@ -31,6 +33,16 @@ rustc_index::newtype_index! {
     pub struct SerializedDepNodeIndex {}
 }
 
+const DEP_NODE_SIZE: usize = std::mem::size_of::<SerializedDepNodeIndex>();
+/// Amount of padding we need to add to the edge list data so that we can retrieve every
+/// SerializedDepNodeIndex with a fixed-size read then mask.
+const DEP_NODE_PAD: usize = DEP_NODE_SIZE - 1;
+/// Amount of bits we need to store the number of used bytes in a SerializedDepNodeIndex.
```

Comment:
```suggestion
/// Number of bits we need to store the number of used bytes in a SerializedDepNodeIndex.
```

Reply:
I was trying to avoid saying "number" twice in a sentence, but that construction reads well to you I agree that "amount" is a bit of an odd word to use in this context.